scripts/consensus_merge.py

```diff
@@ -527,6 +502,40 @@ def build_output_record(single_caller_variants, output_vcf, sample_names, hotspo
 
     return output_record
 
+def allowed_contigs_from_bed(bed_path):
+    """ Gets list of contigs from BED file
+        File must be tab-delimited with contig names in first field
+        Order of contigs in file will be preserved in consensus output
+
+        Args:
+            bed_path (str): path to BED file
+
+        Raises:
+            IOError if bed_path does not point to a file
+                    if no contig names are identified
+                    if contig names are not unique in file
```

Comment:
That would fail in this version. The reason I didn't write it to work the way you had in mind was to preserve the BED order of the contigs in the output. I can change it so that it doesn't require uniqueness and would retain the order of first occurrence of a contig name in the BED file. Would that work for you?

A BED file isn't really necessary since it doesn't use the coordinate information, but if there is a particular such file you had in mind for this purpose, ok.

Reply:
Yes, sounds good. I will also try to incorporate the suggestion attached to Dan's comment about rejecting with an error user-submitted contig names that are not in the Strelka2 header.